definitions/src/lib.rs

```diff
@@ -16,3 +16,6 @@ pub const MEMORY_FRAME_SHIFTS: usize = 18;
 pub const MEMORY_FRAMESIZE: usize = 1 << MEMORY_FRAME_SHIFTS;
 pub const MEMORY_FRAMES: usize = RISCV_MAX_MEMORY / MEMORY_FRAMESIZE;
 pub const MEMORY_FRAME_PAGE_SHIFTS: usize = MEMORY_FRAME_SHIFTS - RISCV_PAGE_SHIFTS;
+
+pub const ISA_IMAC: u8 = 0b0000_0000;
```

Comment:
Let's be precise here, the model we actually implemented is `RV64IMC`, not `IMAC`

Reply:
I will correct it.